src/checkPassword.test.js

```diff
@@ -8,12 +8,33 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(checkPassword).toBeTruthy();
```

Comment:
```suggestion
    expect(checkPassword).toBeTruthy();
```
you should check that this is a boolean. doesn't matter truthy or falsy